client/util.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/drand/drand/log"
 )
 
+// How long to wait after the period begins to ask for the randomness from that period.
+// TODO: ongoing re-calibration of this value.
+const slack = time.Second
```

Comment:
Can you recall again why we aren't using the stream function instead of manually giving some slack ?

Reply:
because you can't push over http.